python/pyarrow/lib.pxd

```diff
@@ -531,6 +531,7 @@ cdef class RecordBatch(_Tabular):
         Schema _schema
 
     cdef void init(self, const shared_ptr[CRecordBatch]& table)
+    cdef void _assert_cpu(self) except *
```

Comment:
So the conclusion is that cython still needs this one, even though it is declared on the base class as well?